spanner/value.go

```diff
@@ -3471,6 +3471,30 @@ func encodeValue(v interface{}) (*proto3.Value, *sppb.Type, error) {
 			}
 		}
 		pt = listType(intType())
+	case uint:
```

Comment:
nit: shouldnt' we add support for uint type in `decodeValue` method also?

Reply:
+1

Reply:
gorm.Model tests were able decode to uint with making any changes in `decodeValue` function, seems like `default` case is sufficient, let me add tests validating it